pytest_fixtures/api_fixtures.py

```diff
@@ -637,6 +637,8 @@ def setting_update(request):
     """
     setting_object = entities.Setting().search(query={'search': f'name={request.param}'})[0]
     default_setting_value = setting_object.value
+    if default_setting_value is None:
```

Comment:
You could also express this as `default_setting_value = setting_object.value or ''`

Reply:
Wow, I didn't know that was a thing. That's cool, thanks for enlightening me! Updated.